packages/keyring-api/src/api/caip.test.ts

```diff
@@ -1,4 +1,40 @@
-import { isCaipAssetId, isCaipAssetType } from './caip';
+import { isCaipAssetId, isCaipAssetType, isCaipChainId } from './caip';
+
+describe('isCaipChainId', () => {
+  // Imported from: https://github.com/ChainAgnostic/CAIPs/blob/main/CAIPs/caip-2.md#test-cases
+  it.each([
+    'eip155:1',
+    'bip122:000000000019d6689c085ae165831e93',
+    'bip122:12a765e31ffd4059bada1e25190f6e98',
+    'bip122:fdbe99b90c90bae7505796461471d89a',
+    'cosmos:cosmoshub-2',
+    'cosmos:cosmoshub-3',
+    'cosmos:Binance-Chain-Tigris',
+    'cosmos:iov-mainnet',
+    'starknet:SN_GOERLI',
+    'lip9:9ee11e9df416b18b',
+    'chainstd:8c3444cf8970a9e41a706fab93e7a6c4',
+  ])('returns true for a valid chain id %s', (id) => {
+    expect(isCaipChainId(id)).toBe(true);
+  });
+
+  it.each([
+    true,
+    false,
+    null,
+    undefined,
+    1,
+    {},
+    [],
+    '',
+    '!@#$%^&*()',
+    'foo',
+    'eip155',
+    'eip155:',
+  ])('returns false for an invalid chain id %s', (id) => {
+    expect(isCaipChainId(id)).toBe(false);
+  });
+});
```

Comment:
I re-exported this because of code coverage. I was not able to "skip" the re-exported types from it.

But we definitely needs to remove those tests now that we are using `@metamask/utils`!

Reply:
See:
- https://github.com/MetaMask/utils/pull/228

Reply:
Now available on `@metamask/utils@11.10`